packages/web-app-files/src/components/FilesList/Pagination.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <oc-pagination
+    v-if="pages > 1"
+    :pages="pages"
+    :current-page="currentPage"
+    :max-displayed="3"
+    :current-route="targetRoute"
+    class="files-pagination uk-flex uk-flex-center oc-my-s"
+  />
+</template>
+
+<script>
+import { mapState, mapGetters } from 'vuex'
+
+export default {
+  computed: {
+    ...mapState('Files', ['currentPage']),
+    ...mapGetters('Files', ['pages']),
+
+    targetRoute() {
+      return { name: this.$route.name, query: this.$route.query, params: this.$route.params }
```

Comment:
Why not just `return this.$route`? 🤔

Reply:
Good question :see_no_evil: I think I had some strange reason when implementing it originally but it works fine when testing with the `$route` so I pushed it in new commit.